packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -173,33 +280,33 @@ function generateReleaseNoteSlug(node) {
 }
 
 // https://www.gatsbyjs.org/docs/mdx/programmatically-creating-pages/#create-pages-from-sourced-mdx-files
-exports.createPages = async ({ graphql, actions, reporter }, pluginOptions) => {
-  const allContentPagesResult = await graphql(`
+exports.createPages = async (...args) => {
+  await createContentPages(...args);
+  await createReleaseNotePages(...args);
+};
+
+async function createContentPages(
+  { graphql, actions, reporter },
+  pluginOptions
+) {
+  const result = await graphql(`
     query QueryAllContentPages {
-      contents: allFile(
-        filter: {
-          sourceInstanceName: { eq: "content" }
-          internal: { mediaType: { eq: "text/mdx" } }
-        }
-      ) {
+      allContentPage {
         nodes {
-          childMdx {
-            id
-            fields {
-              slug
-              title
-              beta
-              isGlobalBeta
-              excludeFromSearchIndex
-            }
-          }
-          name
+          slug
         }
       }
+      allReleaseNotePage {
+        totalCount
+      }
```

Comment:
We can use this to determine if there are any release notes in the website.

We can then be more specific and define some filters to which release notes we want to include. For example, we can exclude all the ones that are not marked as published.